pktgen_s/server/pktgen_server.c

```diff
@@ -0,0 +1,331 @@
+/*
+ * pktgen_server.c
+ * 	
+ * Runs as a daemon and listens to jobs from pktgen_scheduler.py
+ *
+ */
+
```

Comment:
General note on this file: it's got a lot going on. Some of these functions should be pulled into their own files